From da6fda5e476773b8b55a5638435e8740bbee7543 Mon Sep 17 00:00:00 2001
From: Jan Paul <jan.paul@dfki.de>
Date: Fri, 8 Jan 2016 17:00:27 +0100
Subject: changes necessary for general interaction interface


diff --git slam-envire_core/.gitignore slam-envire_core/.gitignore
index 39066cb..42303f3 100644
--- slam-envire_core/.gitignore
+++ slam-envire_core/.gitignore
@@ -5,3 +5,4 @@ build/
 /.idea/
 *.kdev4
 /.settings/
+/CMakeLists.txt~
diff --git slam-envire_core/CMakeLists.txt slam-envire_core/CMakeLists.txt
index aa49d0e..b2b4a7d 100644
--- slam-envire_core/CMakeLists.txt
+++ slam-envire_core/CMakeLists.txt
@@ -6,7 +6,7 @@ find_package(Rock)
 set(ROCK_TEST_ENABLED ON)
 
 option(COVERAGE "Enable code coverage. run 'make test && make coverage' to generate the coverage report. The report will be in ${CMAKE_BINARY_DIR}/cov" OFF)
-option(ENABLE_PLUGINS "Enable the plugin system. Disable this to get rid of the dependency to the class_loader" ON)
+option(ENABLE_PLUGINS "Enable the plugin system. Disable this to get rid of the dependency to the class_loader" OFF)
 
 if(ENABLE_PLUGINS)
   #this definition is used in the source to include/exclude the plugin headers
diff --git slam-envire_core/src/CMakeLists.txt slam-envire_core/src/CMakeLists.txt
index 9ffecbd..60b8306 100644
--- slam-envire_core/src/CMakeLists.txt
+++ slam-envire_core/src/CMakeLists.txt
@@ -1,4 +1,5 @@
-find_package(Boost COMPONENTS serialization)
+#find_package(Boost COMPONENTS serialization)
+find_package(Boost COMPONENTS serialization system filesystem thread)
 
 
 set(headers items/ItemBase.hpp
